app/controllers/api/v1/mattermost_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::MattermostController < Api::ApplicationController
+  before_action :authenticate_mattermost
+
+  def create
+    if params[:command]
+      render json: Commands.run(command_params).response
```

Comment:
I think it'd be better not to have this coupling between the class names in our code and the settings in our mattermost instance. We should be able to change the trigger word at will (or create an alias) just by updating our mattermost settings.

Can we instead have separate routes for where and whereis, that know which Command subclass to create?